### ISSUE TITLE
Reward model training: No such file or directory: '.cache/augmented_latin_cyrillic_oasst_2023-03-27_v2.jsonl'

### ISSUE THREAD

**Issue by theophilegervet** [LAION-AI/Open-Assistant#3570]
While running reward model training with 
```
python trainer_rm.py --configs defaults_rm oasst-rm-1-pythia-1.4b
```
and no changes to the config, I get the following error
```
OASST HF dataset OpenAssistant/oasst1: len(train)=13540, len(val)=1447
Traceback (most recent call last):
  File "/home/tgervet/Open-Assistant/model/model_training/trainer_rm.py", line 334, in <module>
    main()
  File "/home/tgervet/Open-Assistant/model/model_training/trainer_rm.py", line 193, in main
    train, evals = get_dataset(training_conf, mode="rm")
  File "/home/tgervet/Open-Assistant/model/model_training/utils/utils.py", line 378, in get_dataset
    train, val = get_one_dataset(conf, dataset_name, mode=mode, **kwargs)
  File "/home/tgervet/Open-Assistant/model/model_training/custom_datasets/__init__.py", line 157, in get_one_dataset
    train = AugmentedOA(data_path + "/" + kwargs["input_file_path"], split="train")
  File "/home/tgervet/Open-Assistant/model/model_training/custom_datasets/rank_datasets.py", line 158, in __init__
    with open(json_filename, "r", encoding="utf-8") as f:
FileNotFoundError: [Errno 2] No such file or directory: '.cache/augmented_latin_cyrillic_oasst_2023-03-27_v2.jsonl'
```
Am I missing something? Should I download data first?

**Comment by theophilegervet** [comment]
@theblackcat102 Would you have this `augmented_latin_cyrillic_oasst_2023-03-27_v2.jsonl` file by any chance? @andreaskoepf said you might :)

**Comment by andreaskoepf** [comment]
@theophilegervet Our released RM [oasst-rm-2-pythia-6.9b-epoch-1](https://huggingface.co/OpenAssistant/oasst-rm-2-pythia-6.9b-epoch-1) which were also used for the RLHF tuning was not trained on the augmented data. The augmentation was an experiment to add several broken/very low quality replies, it is not really necessary. If you want to run first RM experiments you can use the file `2023-04-12_oasst_ready.trees.jsonl.gz` of [OASST1](https://huggingface.co/datasets/OpenAssistant/oasst1).
